package.json

```diff
@@ -28,6 +28,7 @@
     "react-scripts": "4.0.3",
     "react-test-renderer": "^17.0.2",
     "redux": "^4.1.1",
+    "redux-logger": "^3.0.6",
```

Comment:
- [ ] You seem to have added more rules to the eslint config file kindly remove them and ensure you use the rules stated [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).